bitcoin/src/blockdata/locktime/relative.rs

```diff
@@ -191,6 +338,31 @@ impl fmt::Display for LockTime {
     }
 }
 
+impl convert::TryFrom<Sequence> for LockTime {
+    type Error = DisabledLockTimeError;
+    fn try_from(seq: Sequence) -> Result<LockTime, DisabledLockTimeError> {
+        LockTime::from_sequence(seq)
+    }
+}
+
+impl From<LockTime> for Sequence {
+    fn from(lt: LockTime) -> Sequence { lt.to_sequence() }
+}
+
+/// Error returned when a sequence number is parsed as a lock time, but its
+/// "disable" flag is set.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct DisabledLockTimeError(u32);
```

Comment:
This is a leaf error type so we should probably provide a getter if I remembered correctly and that is the policy we settled upon.